src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/TagletWriter.java

```diff
@@ -258,111 +259,119 @@ public static void genTagOutput(TagletManager tagletManager,
+     * @return the generated content
      */
-    public abstract Content commentTagsToOutput(DocTree holderTag, List<? extends DocTree> tags);
+    public abstract Content commentTagsToOutput(DocTree holderTree, List<? extends DocTree> trees);
 
     /**
-     * Converts inline tags and text to TagOutput, expanding the
+     * Converts inline tags and text to content, expanding the
      * inline tags along the way.  Called wherever text can contain
      * an inline tag, such as in comments or in free-form text arguments
-     * to non-inline tags.
+     * to block tags.
+     *
+     * @param element The element that owns the documentation
+     * @param trees  list of {@code DocTree} nodes containing text and inline tags (often alternating)
+     *               present in the text of interest for this doc
      *
-     * @param holder the element where comment resides.
-     * @param tags   array of text tags and inline tags (often alternating)
-     *               present in the text of interest for this doc.
-     * @return the {@link Content} representing the comments.
+     * @return the generated content
      */
-    public abstract Content commentTagsToOutput(Element holder, List<? extends DocTree> tags);
+    public abstract Content commentTagsToOutput(Element element, List<? extends DocTree> trees);
 
     /**
-     * Converts inline tags and text to TagOutput, expanding the
+     * Converts inline tags and text to content, expanding the
      * inline tags along the way.  Called wherever text can contain
      * an inline tag, such as in comments or in free-form text arguments
      * to non-inline tags.
      *
-     * @param holderTag the tag that holds the documentation.
-     * @param holder the element where comment resides.
-     * @param tags   array of text tags and inline tags (often alternating)
-     *               present in the text of interest for this doc.
-     * @param isFirstSentence true if this is the first sentence.
-     * @return the {@link Content} representing the comments.
+     * @param element         the element where comment resides
+     * @param holder          the tag that holds the documentation
+     * @param trees           array of text tags and inline tags (often alternating)
+     *                        present in the text of interest for this doc
+     * @param isFirstSentence true if this is the first sentence
+     *
+     * @return the generated content
      */
-    public abstract Content commentTagsToOutput(DocTree holderTag,
-        Element holder, List<? extends DocTree> tags, boolean isFirstSentence);
+    public abstract Content commentTagsToOutput(Element element, DocTree holder,
+                                                List<? extends DocTree> trees, boolean isFirstSentence);
 
     /**
-     * @return an instance of the configuration used for this doclet.
+     * Returns  an instance of the configuration used for this doclet.
```

Comment:
Remove extra whitespace in "Returns  an".

Reply:
👍